volume/local/local.go

```diff
@@ -366,3 +366,15 @@ func getAddress(opts string) string {
 	}
 	return ""
 }
+
+// getPassword finds out a password from options
+func getPassword(opts string) string {
+	optsList := strings.Split(opts, ",")
+	for i := 0; i < len(optsList); i++ {
+		if strings.HasPrefix(optsList[i], "password=") {
+			passwd := strings.SplitN(optsList[i], "=", 2)[1]
+			return passwd
+		}
+	}
+	return ""
+}
```

Comment:
I was concerned about this implementation, but in `mount.cifs` we have the following delightful gem: :see_no_evil: :heart:

```
              Note  that  a  password  which  contains the delimiter character
              (i.e. a comma ',') will fail to be parsed correctly on the  com‐
              mand  line. However, the same password defined in the PASSWD en‐
              vironment variable or via a credentials file (see below) or  en‐
              tered at the password prompt will be read correctly.
```